src/stactools/modis/fragments/MOD10A2/061/bands.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "name": "Maximum_Snow_Extent",
+        "description": "Maximum snow extent over the 8-day period."
+    },
+    {
+        "name": "Eight_Day _Snow_Cover",
```

Comment:
```suggestion
        "name": "Eight_Day_Snow_Cover",
```

Reply:
Removed space in name for "bands.json" on MOD10A2, MOD13A1, MOD13Q1, MYD10A2, MYD13A1, and MYD13Q1.